include/SugarFields/Fields/File/ListView.tpl

```diff
@@ -48,4 +48,7 @@
 {/capture}
 {if strlen($imageURL)>1}{sugar_getimage name=$imageName alt=$imageName other_attributes='border="0" '}{/if}
 {/if}
-</a>
\ No newline at end of file
+</a>&nbsp;
+<a href="index.php?preview=yes&entryPoint=download&id={$parentFieldArray.ID}&type={$displayParams.module}{$vardef.displayParams.module}" class="tabDetailViewDFLink" target='_blank' style="border-bottom: 0px;">
+	<i class="fas fa-eye"></i>
```

Comment:
Missing Icon, Please consider changing the css class to "glyphicon glyphicon-eye-open".

Also I think it might be better to create a send link next to this one with the eye. So that users can choose to either download it or preview it.